Add additional info to the README

chasenlehara opened the pull request:
This adds more info about setting up the repo locally to the README.

All of the “Setup” and “Running the app” instructions can be followed to get the app running in the Android Emulator.

[Rendered file.](https://github.com/bitovi/react-native-pmo/blob/readme/README.md)

DanDaManTran commented on README.md:
```diff
@@ -1,34 +1,167 @@
+## Running the app
 
-Install Expo Go on your mobile device from the appropriate app store. This will allow you to quickly run demos you find online with your mobile device. This app will not run in Expo Go because of OAuth, but many other apps will so its a useful tool to be aware of.
+Build and launch the app in the Android Emulator with:
```
I would hint, or inform the user that they need the android studio emulator running before 'npm run android'

chasenlehara replied:
It’s here:

>  - Using a virtual device (under Preparing the Android device)

Point taken though, that should probably be moved from the Setup instructions to the commands for running.

DanDaManTran reviewed (changes requested):
Just a minor update

chasenlehara committed:
Add additional info to the README

This adds more info about setting up the repo locally to the README.

All of the “Setup” and “Running the app” instructions can be followed to get the app running in the Android Emulator.
